client/client.go

```diff
@@ -33,20 +38,131 @@ import (
 	"github.com/transparency-dev/trillian-tessera/api"
 	"github.com/transparency-dev/trillian-tessera/api/layout"
 	"golang.org/x/mod/sumdb/note"
+	"k8s.io/klog/v2"
 )
 
 var (
 	hasher = rfc6962.DefaultHasher
 )
 
-// Fetcher is the signature of a function which can retrieve arbitrary files from
-// a log's data storage, via whatever appropriate mechanism.
-// The path parameter is relative to the root of the log storage.
+// CheckpointFetcherFunc is the signature of a function which can retrieve the latest
+// checkpoint from a log's data storage.
 //
 // Note that the implementation of this MUST return (either directly or wrapped)
 // an os.ErrIsNotExit when the file referenced by path does not exist, e.g. a HTTP
```

Comment:
```suggestion
// an os.ErrIsNotExist when the file referenced by path does not exist, e.g. a HTTP
```

Existing typo, but I just spotted it now.

passim.

Reply:
done